app/FAIRshakeWeb/view.py

```diff
@@ -43,8 +55,129 @@ def register():
     current_user={},
   )
 
-@app.route('/login', methods=['GET'])
-def login():
-  return render_template('login.html',
+@app.route('/project/<int:project>/resources', methods=['GET'])
+def resources(repository: RepositoryAPI, assessment: AssessmentAPI, score: ScoreAPI, project):
+  resources=repository.get(tag=project)
+  current_user_assessments = []
+  assessment_count = {}
+  aggregate_scores = {}
+  for resource in resources: #resource-id=1-7
+    try:
+      assessment.get(object=resource.id, user=current_user['id']) ### redesign, if there is an assessment for this object for this user
+      current_user_assessments.append(resource.id) # just array with resource ids
```

Comment:
In general, appending to a list is almost never required in python.

Use instead:
```python
current_user_assessments = [
  assessment.get(object=resource.id, user=current_user.id)
  for resource in resources
]
```

Reply:
```
current_user_assessed_resources = [
    resource.id for resource in resources
    if assessment.get(object=resource.id, user=current_user['id'])
  ]
```
?